app/build.gradle

```diff
@@ -98,17 +98,17 @@ dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar", "*.aar"], exclude: "proguard-base-7.2.2.jar")
 
     implementation "androidx.appcompat:appcompat:1.6.1"
-    implementation "androidx.recyclerview:recyclerview:1.2.1"
+    implementation "androidx.recyclerview:recyclerview:1.3.0"
     implementation "androidx.cardview:cardview:1.0.0"
     implementation "androidx.viewpager:viewpager:1.0.0"
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.1.0"
-    implementation "com.google.android.material:material:1.8.0"
+    implementation "com.google.android.material:material:1.9.0"
     implementation "com.airbnb.android:lottie:3.1.0"
     implementation "com.google.android.flexbox:flexbox:3.0.0"
 
-    implementation "androidx.annotation:annotation:1.5.0"
-    implementation "com.android.tools:annotations:30.4.1"
-    implementation "com.google.guava:guava:31.1-jre"
+    implementation "androidx.annotation:annotation:1.6.0"
+    implementation "com.android.tools:annotations:31.0.2"
+    implementation "com.google.guava:guava:32.0.1-jre"
```

Comment:
cmon at least make a separate commit for this

<sub><sup>not saying you should fix that rn, just don't in the future</sup></sub>

Reply:
okay